lib/README.md

```diff
@@ -0,0 +1,72 @@
+# Nixpkgs lib
+
+This folder contains the implementation, documentation and tests for the Nixpkgs `lib` library.
```

Comment:
```suggestion
This directory contains the implementation, documentation and tests for the Nixpkgs `lib` library.
```

This is completely minor, but using "directory" consistently would probably help beginners in particular.